### PR TITLE
swift: adjust ring-file output parsing for Newton

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by jsuchome** [comment]
... I mean: it worked for me when I manually applied it and redeployed swift. But here it looks that gating does not pass anyway

File: chef/cookbooks/swift/providers/ringfile.rb [hunk 1]
```diff
@@ -112,9 +112,9 @@ def parse_dev_info_line(line, ringinfo)
   Chef::Log.debug("parsing dev info: " + line)
 
   # Line looks like this:
-  #   id  region  zone      ip address  port  replication ip  replication port      name weight partitions balance meta
-  #   0       1     0  192.168.125.14  6000  192.168.125.14              6000 2d4dc9923ed244dc9cac8f283ca79748  99.00          0 -100.00
-  line =~ /^\s+(\d+)\s+(\d+)\s+(\d+)\s+(\d+\.\d+\.\d+\.\d+)\s+(\d+)\s+(\d+\.\d+\.\d+\.\d+)\s+(\d+)\s+(\S+)\s+([0-9.]+)\s+(\d+)\s*([-0-9.]+)\s*$/
+  # id region zone  ip address:port      replication ip:port  name                              weight partitions balance meta
+  #  0      1    0  192.168.125.14:6000  192.168.125.14:6000  2d4dc9923ed244dc9cac8f283ca79748  99.00           0 -100.00
+  line =~ /^\s+(\d+)\s+(\d+)\s+(\d+)\s+(\d+\.\d+\.\d+\.\d+)[:\s+](\d+)\s+(\d+\.\d+\.\d+\.\d+)[:\s+](\d+)\s+(\S+)\s+([0-9.]+)\s+(\d+)\s*([-0-9.]+)\s*$/
```
Comment (vuntz) [inline chef/cookbooks/swift/providers/ringfile.rb]: Stupid question, but why `[:\s+]` and not simply `:\s+`?

Comment (vuntz) [inline chef/cookbooks/swift/providers/ringfile.rb]: Ah, got it, to stay compatible with old versions.

**Comment by dirkmueller** [comment]
[(5) S53P4 switch swift to Newton version](https://trello.com/c/IywUtcYv/430-5-s53p4-switch-swift-to-newton-version)

**Pull request merged**